weave/guides/integrations/anthropic.mdx

```diff
@@ -1,24 +1,22 @@
 ---
 title: "Anthropic"
-description: "Use Weave to automatically track and log LLM calls made via the Anthropic Python library"
+description: "Use Weave to automatically track and log LLM calls made via the Anthropic SDK"
 ---
 
 <a target="_blank" href="https://colab.research.google.com/github/wandb/examples/blob/master/weave/docs/quickstart_anthropic.ipynb">
   <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
 </a>
 
-Weave automatically tracks and logs LLM calls made via the [Anthropic Python library](https://github.com/anthropics/anthropic-sdk-python), after `weave.init()` is called.
-
-<Note>
-Do you want to experiment with Anthropic models on Weave without any set up? Try the [LLM Playground](../tools/playground).
-</Note>
+When you integrate Weave into your code, it automatically tracks and logs LLM calls made with the Anthropic SDK for both [Python](https://github.com/anthropics/anthropic-sdk-python) and [TypeScript](https://github.com/anthropics/anthropic-sdk-typescript). Weave does this by automatically invoking Anthropic's `Messages.create` method.
 
 ## Traces
 
-It’s important to store traces of LLM applications in a central database, both during development and in production. You’ll use these traces for debugging, and as a dataset that will help you improve your application.
+Weave automatically captures traces for the Anthropic SDK when you add `weave.init("your-team-name/your-project-name")` to your code. If you don't specify a team name as an argument in `weave.init()`, Weave logs output to your [default W&B entity](/platform/app/settings-page/user-settings/#default-team). If you don't specify a project name, Weave fails to initialize.
```

Comment:
I did not know that. In W&B Models, logging a run without a specified project goes to an "Uncategorized" project.